virttest/libvirt_xml/vm_xml.py

```diff
@@ -3325,3 +3333,39 @@ def __init__(self, virsh_instance=base.virsh):
                                attribute='unit')
         super(VMFeaturesHptXML, self).__init__(virsh_instance=virsh_instance)
         self.xml = '<hpt/>'
+
+
+class VMKeywrapXML(base.LibvirtXMLBase):
+    """
+    Keywrap class for s390x ciphers on QEMU
+    xpath: /domain/keywrap
```

Comment:
It is better add one simple example show how VMKeywrapXML looks like

Reply:
@chunfuwen Do you mean doc string should contain 
```xml
<keywrap>
  <cipher name='aes' state='on'/>
</keywrap>
```
or do you mean an actual usage in code like e.g. https://github.com/autotest/tp-libvirt/blob/e0d140e83b90505c5de25e73545317d7126f6f3a/libvirt/tests/src/libvirt_keywrap.py#L29 ?

Reply:
I   mean doc string better contain sample VMKeywrapXML

Reply:
> I mean doc string better contain sample VMKeywrapXML

Got it. So for example the following?
```python
class VMKeywrapXML(base.LibvirtXMLBase):
    """
    Keywrap class for s390x ciphers on QEMU
    xpath: /domain/keywrap

    Example:
    
        vmxml = VMXML.new_from_dumpxml(vm_name)
        kw = VMKeywrapXML()
        kw.set_cipher("aes", "off")
        vmxml.set_keywrap(kw)

    :param ....
```